quetz/authorization.py

```diff
@@ -285,6 +285,9 @@ def assert_set_channel_size_limit(self):
             msg="only server maintainer or owner can set channel size limit",
         )
 
+    def assert_channel_db_cleanup(self, channel_name):
+        self.assert_channel_roles(channel_name, [OWNER, MAINTAINER])
```

Comment:
I think only OWNER should be allowed here (maintainer might not be able to see the logs of the serveretc).

Reply:
Done.